src/grisp_connect.app.src

```diff
@@ -7,6 +7,7 @@
         kernel,
         inets,
         stdlib,
+        public_key,
```

Comment:
This is already started by `grisp_cryptoauth`

Reply:
But dialyser is complaining if I don't add it there.

Reply:
maybe add grisp_cryptoauth to the plts ?

Reply:
As grisp_connect is using directly public_key now, I still think it makes sense to add it there. What if girsp_cryptoauth stopped depending it ?